.github/workflows/test.yml

```diff
@@ -8,6 +8,17 @@ on:
       - "**"
 
 jobs:
+  run-adminUi-checks:
+    runs-on: ubuntu-latest
+    name: AdminUi Run Checks
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install script dependencies
+        run: npm install --prefix ./.ci
+      - name: Run Checks
+        run: bash AdminUi/src/AdminUi/ClientApp/.ci/runChecks.sh
```

Comment:
I would probably move the script to the .ci folder, call it runAdminUiChecks. Then, in the first line of the script, save the current working directory (`$INITIAL_DIR=$pwd`), switch to the directory you need to be in, run the commands, and then switch back to the `INITIAL_DIR`.